src/eckit/types/Time.h

```diff
@@ -32,9 +32,9 @@ typedef double Second;
 class Time {
 
 public:  // methods
-    Time(long, long, long);
-    Time(long seconds = 0);
-    Time(const std::string&);
+    Time(long, long, long, bool extended = false);
+    Time(long seconds = 0, bool extended = false);
+    Time(const std::string&, bool extended = false);
```

Comment:
Missing the arguments names (since they are not deducible by the type).

To be consistent, it's also missing one constructor
    Time(long minutes, long seconds, bool extended = false);